### DISCUSSION TITLE
"Server error"  I can't receive the email confirming my registration

### DISCUSSION THREAD

**Discussion by WhiteRabbit21m** [simple-login/app#1779] [Self-hosting question]
hello, I installed version 4.6.2 but I can not receive the confirmation email and then do my login. I also tried to see if everything worked to your instructions and I can receive the test email on my domain. Does anyone have any idea why I am getting this error? " Server error
Looks like we are having some server issues...

We are notified and will look at this issue asap! "
thanks to those who will give me a hand to understand.

**Comment by WhiteRabbit21m** [comment]


**Reply by t2noob** [reply]
Just to make sure it isn't a simple mistake did you enable registration? You have to leave it on make the account and then turn off registration?.

Can you send email with swaks?  

Have you gone through the troubleshooting page see if it catches anything?

**Reply by WhiteRabbit21m** [reply]
Hello, registration is enabled and I can receive the email tests with Swaks. I can't find a discussion that suits me on the troubleshooting page.

**Reply by t2noob** [reply]
Youve gone through this trouble shooting guide? Sounds like you're having the very first issue of not receiving the welcome email. 
 https://github.com/simple-login/app/blob/master/docs/troubleshooting.md

**Reply by WhiteRabbit21m** [reply]
That's exactly the problem. But if I try to check with swaks I can get the email. That's why I can't figure out where the problem is.
